mlflow/cli.py

```diff
@@ -121,6 +121,10 @@ def run(uri, entry_point, version, param_list, experiment_name, experiment_id, b
         except ValueError as e:
             eprint("Invalid cluster spec JSON. Parse error: %s" % e)
```

Comment:
Could we refactor this line to say `Invalid backend config JSON. Parse error...`?

Reply:
done!